scrapers/FemdomEmpire.yml

```diff
@@ -0,0 +1,66 @@
+name: "FemdomEmpire"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/trailers/
+      - feminized.com/tour/trailers/
+    scraper: sceneScraper
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - femdomempire.com/tour/models/
+      - feminized.com/tour/models/
+    scraper: performerScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class="videoDetails clear"]/h3
+      Date:
+        selector: //p[span[contains(.,"Date Added")]]
+        postProcess:
+          - replace:
+            - regex: "Date Added:"
+              with:
+          - parseDate: January 2, 2006
+      Details: //div[@class="videoDetails clear"]/p
+      Performers:
+        Name: //li[@class="update_models"]/a
+      Tags:
+        Name: //ul/li[@class="label" and contains(.,"Categories:")]/..//a
+      Studio:
+        Name:
+          selector: //div[@id='logo']/h1/a/span
+      Image:
+        selector: //base/@href|//img[@class="update_thumb thumbs stdimage"]/@src0_1x
+        concat: "|"
+        postProcess:
+          - replace:
+            - regex: "(^[^|]+)\\|/tour/([^\\.]+\\.jpg).*"
+              with: $1$2
+  performerScraper:
+    common:
+      $performerInfo: "//div[@class='contentBox'][1]"
+    performer:
+      Name:
+        selector: "$performerInfo//div[@class='profile-details']/h3"
+      Gender:
+        selector: "$performerInfo/div[@class='title']/h3"
+        postProcess:
+          - replace:
+            - regex: "^Mistress Sissy .*$"
+              with: Male
+          - replace:
+            - regex: "^Mistress (.*)$"
+              with: Female
+      Image:
+        # we use the union | and concat to get the base url for the image
+        selector: //base/@href|(//div[@class="flexslider-carousel"]//img/@src0_1x)[1] 
+        concat: "|"
+        postProcess:
+          - replace:
+            - regex: "(^[^|]+)\\|/tour/([^\\.]+\\.jpg).*"
+              with: $1$2
+
+
+# Last Updated October 10, 2020
```

Comment:
A last change to update the `Last Updated` date to today

Reply:
🤦‍♀️  I feel a fool now

Reply:
it happens to all of us :-P